src/vs/workbench/electron-browser/commands.ts

```diff
@@ -371,6 +371,17 @@ export function registerCommands(): void {
 		}
 	});
 
+	KeybindingsRegistry.registerCommandAndKeybindingRule({
```

Comment:
I don't think this is the right way to handle this, it would be better if there was an explicit exception on Windows similar to this:

https://github.com/Microsoft/vscode/blob/851d231fe94f45cee44f5ddaa0b2a89fa3c8df96/src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts#L328-L330

We don't want alt+f4 handled by VS Code's keybinding system, it would be better to lean on the native keybinding. Plus we will always want alt+f4 to quit on Windows if it's used, I don't see why anyone would/should configure this.